pom.xml

```diff
@@ -1175,6 +1175,11 @@
                 <artifactId>spring-boot-starter-webflux</artifactId>
                 <version>${spring-boot.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-starter-data-mongodb</artifactId>
+                <version>${spring-boot.version}</version>
+            </dependency>
```

Comment:
DAO'ya eklemişsin zaten, burada da olması gerekiyor mu?

Reply:
Thingsboard yapısında tüm dependency'ler sürümleri ile birlikte ana pom.xml'e ekleniyor ve tüm sub-projeler ana pom.xml'i import edip dependency adı ile gerekli dependency'leri alıyor. Buradan kaldırıp DAO pom.xml'de sürüm vererek alınabilir ama bu sayede tüm dependency'ler doğru sürümler ile bir yerde eklenebiliyor. Bu [multi module maven tutorial](https://www.baeldung.com/maven-multi-module#3-enable-dependency-management-in-parent-project)'ın açıklamasına bakabilirsiniz.